.travis.yml

```diff
@@ -0,0 +1,12 @@
+os: linux
+language: php
+dist: bionic
+php:
+  - 7.3
+  - 7.4
+install:
+  - composer install
+script:
```

Comment:
I think we should add the linter stuff as well, since I would like to not have a PSM deja vu.

Reply:
ok ok will add a linter

Reply:
Nvm, this is the auto0 package, I care less now.

Reply:
Added it anyway